package.json

```diff
@@ -61,7 +61,8 @@
 		"test-gui": "NODE_ENV=test NODE_PATH=tests:_inc/client tests/runner.js gui",
 		"test-modules": "NODE_ENV=test NODE_PATH=tests:_inc/client tests/runner.js modules",
 		"test-dangerci-and-adminpage-and-extensions": "yarn danger ci && yarn test-adminpage && yarn test-extensions",
-		"reformat-files": "./node_modules/.bin/prettier --ignore-path .eslintignore --write \"**/*.{js,jsx,json}\""
+		"reformat-files": "./node_modules/.bin/prettier --ignore-path .eslintignore --write \"**/*.{js,jsx,json}\"",
+		"packages:add-textdomain": "wpi18n addtextdomain --textdomain=jetpack --glob-pattern='packages/**/*.php'"
```

Comment:
I think we want to do it on the `vendor` folder as that's the one that carries the packages when we bundle/ship the package.

Reply:
I think we can just remove vendor from the blacklist here: `"add-textdomain": "wpi18n addtextdomain --textdomain=jetpack --glob-pattern='!(docker|node_modules|tests|tools|vendor){*.php,**/*.php}'",`

Reply:
Cool. I'll take care of this tomorrow morning.